add-on/chrome/bootstrap.js

```diff
@@ -875,6 +875,12 @@ function shutdown() {
   // Stop waiting for browser windows to open.
   wm.removeListener(WindowListener);
 
+  // If the app is shutting down, don't worry about cleaning up, just let
+  // it fade away...
+  if (reason == 2) {
```

Comment:
You can use the magic global `APP_SHUTDOWN` instead of `2` (and update add-on/chrome/.eslintrc)